internal/devpkg/package.go

```diff
@@ -145,6 +145,7 @@ func newPackage(raw string, isInstallable func() bool, locker lock.Locker) *Pack
 	// assume a Devbox package.
 	parsed, err := flake.ParseInstallable(raw)
 	if err != nil || pkgtype.IsAmbiguous(raw, parsed) {
+		// TODO: This sets runx packages as devbox packages. Not sure if that's what we want.
```

Comment:
oooh :D
good thing runx isn't public yet, so we can land this

Reply:
This hasn't broken anything for us (yet) even though we use runx. But I think in the future this confusion could lead to bugs.